src/reports/package/accessibilityInsightsReport.d.ts

```diff
@@ -47,16 +47,50 @@ declare namespace AccessibilityInsightsReport {
     };
 
     export type SummaryReportParameters = {
-        serviceName: string;
-        axeVersion: string;
-        userAgent: string;
-        crawlDetails: CrawlSummaryDetails;
-        results: SummaryScanResults;
+        serviceName: string,
+        axeVersion: string,
+        userAgent: string,
+        crawlDetails: CrawlSummaryDetails,
+        results: SummaryScanResults,
     };
 
+    export type AxeRuleData = {
+        ruleId: string,
+        tags: string[],
+        description: string,
+        ruleUrl: string
+    }
+
+    export type InstanceFailureData = {
+        urls: string[],
+        elementSelector: string,
+        snippet: string,
+        fix: string,
+    }
+
+    export type RuleFailureData = {
+        rule: AxeRuleData,
+        failures: InstanceFailureData[],
+    }
+
+    export type CombinedReportResults = {
+        failedRules: RuleFailureData[],
+        passedRules: AxeRuleData[],
+        notApplicableRules: AxeRuleData[],
+    }
+
+    export type CombinedReportParameters = {
+        serviceName: string,
+        axeVersion: string,
+        userAgent: string,
+        crawlDetails: CrawlSummaryDetails,
```

Comment:
we probably should rename this `crawlDetails: CrawlSummaryDetails` to something that has no crawl in it like `scanDetails` or similar. We do not crawl for this report data generation